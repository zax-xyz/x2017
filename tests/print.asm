FUNC LABEL 0
    MOV STK A VAL 5
    MOV REG 0 VAL 12
    PRINT STK A
    PRINT VAL 5
    PRINT REG 0
    MOV REG 1 VAL 16
    MOV REG 2 REG 1
    PRINT REG 1
    PRINT REG 2
    RET
