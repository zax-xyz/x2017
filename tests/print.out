5
5
12
16
16
